source/_integrations/ultraloq.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: Ultraloq
+description: Connect and control your Ultraloq Z-Wave series devices using the Z-Wave integration
+ha_release: '2022.11'
+ha_category:
+  - Lock
+ha_domain: ultraloq
+ha_integration_type: virtual
+works_with:
+  - zwave
+ha_iot_standard: zwave
+---
+
+<div class="note warning">
+There is a <a href="https://github.com/zwave-js/node-zwave-js/issues/5445">bug</a> in the firmware of Ultraloq devices that causes the door state sensor to not be updated and the latch to always be reported as "OPEN".<br/>
+We are fully aware that this is not what you expect from a <a href="/blog/2022/07/12/partner-program">Works With Home Assistant</a> brand and we are in contact Ultraloq to get this resolved!
+</div>
+
```

Comment:
```suggestion
```

Reply:
There is a second commit in this PR <https://github.com/home-assistant/home-assistant.io/pull/28728/commits/f365c6c1fa09286d528f4daafa215c9637b1237c> that does exactly that